packages/google_sign_in/google_sign_in_android/README.md

```diff
@@ -13,3 +13,28 @@ should add it to your `pubspec.yaml` as usual.
 
 [1]: https://pub.dev/packages/google_sign_in
 [2]: https://flutter.dev/to/endorsed-federated-plugin
+
+## Integration
```

Comment:
Can we consider linking to the `credentials` docs on this? https://developer.android.com/identity/sign-in/credential-manager-siwg#set-google for the non-firebase option? If it's more confusing than helpful because some steps don't apply, maybe not though.

Reply:
By the time I was updating the READMEs I was so used to skipping down to the code on that page I forgot it had setup instructions! I've adjusted to link to that as the non-Firebase option, and removed some of the details since they are covered more thoroughly there.